spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -516,13 +516,41 @@ describe("Configuration loader", function()
       assert.is_nil(conf)
 
       conf, err = conf_loader(nil, {
-        cassandra_consistency = "FOUR"
+        cassandra_write_consistency = "FOUR"
       })
-      assert.equal("cassandra_consistency has an invalid value: 'FOUR'"
+      assert.equal("cassandra_write_consistency has an invalid value: 'FOUR'"
                  .. " (ALL, EACH_QUORUM, QUORUM, LOCAL_QUORUM, ONE, TWO,"
                  .. " THREE, LOCAL_ONE)", err)
       assert.is_nil(conf)
 
+      conf, err = conf_loader(nil, {
+        cassandra_read_consistency = "FOUR"
+      })
+      assert.equal("cassandra_read_consistency has an invalid value: 'FOUR'"
+                 .. " (ALL, EACH_QUORUM, QUORUM, LOCAL_QUORUM, ONE, TWO,"
+                 .. " THREE, LOCAL_ONE)", err)
+      assert.is_nil(conf)
+
+      conf = assert(conf_loader(nil, {
+        cassandra_consistency = "QUORUM"
+      }))
+      assert.equal("QUORUM", conf.cassandra_read_consistency)
+      assert.equal("QUORUM", conf.cassandra_write_consistency)
+
+      conf = assert(conf_loader(nil, {
+        cassandra_consistency = "QUORUM",
+        cassandra_read_consistency = "TWO"
+      }))
+      assert.equal("TWO", conf.cassandra_read_consistency)
+      assert.equal("QUORUM", conf.cassandra_write_consistency)
+
+      conf = assert(conf_loader(nil, {
+        cassandra_consistency = "QUORUM",
+        cassandra_write_consistency = "LOCAL_QUORUM"
+      }))
+      assert.equal("QUORUM", conf.cassandra_read_consistency)
+      assert.equal("LOCAL_QUORUM", conf.cassandra_write_consistency)
```

Comment:
Please add new tests around the deprecation of properties to a new `describe` block instead. Leave this test unchanged, and add a new `describe` block to this test suite. I did the same in #5771 which isn't merged yet, so you can just add the new `describe` block yourself, in the same place as I added it in #5054: 

```lua
describe("deprecated properties", function()
  it("cassandra_consistency -> cassandra_r/w_consistency", function()

  end)
end)
```

Reply:
@thibaultcha done